tests/tools/fio/main_test.go

```diff
@@ -0,0 +1,19 @@
+package fio
+
+import (
+	"fmt"
+	"os"
+	"testing"
+)
+
+func TestMain(m *testing.M) {
+	Exe := os.Getenv("FIO_EXE")
+	if Exe == "" {
+		fmt.Println("Skipping fio tests if FIO_EXE is not set")
+		os.Exit(0)
```

Comment:
We may want to fail here to avoid inadvertently skipping tests.

Reply:
mm yeah I managed it this way for now to prevent CI from failing until we make the necessary changes to support these tests